### PR TITLE
gh action doc

### PR DESCRIPTION
**Pull request by kalantar**
Documentation for GH action

Signed-off-by: Michael Kalantar <kalantar@us.ibm.com>

### CODE REVIEW

File: docs/tutorials/integrations/ghactions.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+template: main.html
+---
+
+# Use Iter8 in a GitHub workflow
```
Comment (sriumcp) [inline docs/tutorials/integrations/ghactions.md:5]: Could we please say: `Use Iter8 in a GitHub Actions workflow`

**Review by sriumcp — CHANGES_REQUESTED** [review]
minor change requested

File: docs/tutorials/integrations/ghactions.md (hunk 1 above)
Comment (kalantar) [reply docs/tutorials/integrations/ghactions.md]: done

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**